src/modules/metadata/actions.js

```diff
@@ -389,3 +420,29 @@ export const fetchIndividualIfNecessary = individualID => (dispatch, getState) =
     if (individualRecord.isFetching || individualRecord.data) return;  // Don't fetch if already fetching or loaded.
     return dispatch(fetchIndividual(individualID));
 };
+
+
+
+export const fetchPhenopackets = networkAction(() => (dispatch, getState) => ({
+    types: FETCH_PHENOPACKETS,
+    // params: {phenopacketID},
```

Comment:
remove

Reply:
why? it's used in the overview

Reply:
specifically the commented out line, sorry